mlflow/tracking/client.py

```diff
@@ -832,13 +832,14 @@ def set_trace_tag(self, request_id: str, key: str, value: str):
 
         Args:
             request_id: The ID of the trace to set the tag on.
-            key: The string key of the tag. Must be shorter than 250 characters.
-            value: The string value of the tag. Must be shorter than 250 characters.
+            key: The string key of the tag. If than 250 characters, otherwise
+                it will be truncated when stored.
+            value: The string value of the tag. Must be shorter than 250 characters, otherwise
+                it will be truncated when stored.
```

Comment:
nit: `shorter than 250` sounds like the limit is 249. The actual limit is 250, right?

Reply:
Oh true, great catch!